### DISCUSSION TITLE
Marker

### DISCUSSION THREAD

**Discussion by nasriLakhal** [JustFly1984/react-google-maps-api#3237] [Q&A]
i want to change the style of marker to change for an example the direction of an arrow 
since querySelector is not good in react , using useRef instead i didn't find the style to change this 
if there is any other solution you can help me

**Comment by mirhamasala** [comment]
You can use `options` key with `icon`, like so:

```
<MarkerF
  key={spot.slug}
  options={{
    icon: {
      scaledSize: new window.google.maps.Size(56, 56),
      url: spot.category.marker,
    },
  }}
  position={{
    lat: spot.geo.latitude,
    lng: spot.geo.longitude,
  }}
  onClick={() => openInfoWindow(spot)}
  onMouseOver={() => openInfoWindow(spot)}
/>
```
